src/routes/safe/components/Transactions/TxList/TxDetails.tsx

```diff
@@ -111,19 +128,21 @@ export const TxDetails = ({ transaction }: TxDetailsProps): ReactElement => {
   }
 
   return (
-    <TxDetailsContainer>
+    <TxDetailsContainer ownerRows={hasModule ? 3 : 2}>
```

Comment:
Is it necessary to pass an attribute? Can't the component adjust dynamically? Let me know if it was a quick fix because it will eventually be touched in the `TxDetails` redesign task

Reply:
It needs to be passed otherwise the `'tx-owners'` grid does not span to the bottom correctly.

Reply:
Yes, I was thinking if it can adjust dynamically instead of passing how many rows should the grid span.

Reply:
The parent does not have `display: grid`. How would you approach 'stretching' `'tx-owners'`?

Reply:
I see. Thanks for explaining.